src/pymoca/tree.py

```diff
@@ -349,7 +366,19 @@ def find_name(
     if found is not None and rest_of_name:
         found = _find_rest_of_name(found, rest_of_name)
 
-    if copy and isinstance(found, ast.Class):
+    # Maintaining backward compatibility by including InstanceTree (not strictly correct)
```

Comment:
Do you want to make a class of TODOs for cleanups like? When we get rid of the "old" code (I assume that's the goal eventually), we

Reply:
I did not know if this was desired behavior. Many of the examples in `test/models` are written this way. They don't parse in OpenModelica (or probably most other Modelica tools), which *is* a pain.

Can you confirm that you want to break potentially many existing user models? Certainly not to be done before Pymoca 1.0.

Reply:
You have much better read on the current user base, so if you say rip it up in 0.11, I say rip it up in 0.11! After all, they are all 0.* releases!

Reply:
Added the TODO as suggested in faa48be.